.travis.yml

```diff
@@ -0,0 +1,33 @@
+---
+language: go
+
+sudo: required
+
+services:
+  - docker 
+
+notifications:
+  email: false
+
+before_install:
+  - sudo apt-get -qq update
+  - sudo apt-get install -y rpm
+
+script:
+  - GO111MODULE=on go test ./...
+  - go test -race -coverprofile=coverage.txt -covermode=atomic ./...
+  
+
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
+deploy:
```

Comment:
I think you can drop the deploy stanza when it comes to code cov. To configure GitHub releases it's another story and most likely a dedicated PR: you'll need goreleaser.yaml, encrypted GitHub access token, etc.

Reply:
OK